docs/ingress.md

```diff
@@ -0,0 +1,72 @@
+# Ingress
```

Comment:
Is ingress the right title for the document? Seems like you are describing a multi-tier (layered) architecture

https://www.oreilly.com/library/view/software-architecture-patterns/9781491971437/ch01.html

we should probably situate how each specific technology piece fit within presentation/business/services/persistence/database layer.

Reply:
Done

Reply:
file should also be renamed

Reply:
Done

Reply:
I still see ingress.md?

Reply:
@ThomasCardin because this thread I wasn't sure if you pushed all the changes.

Reply:
Yeah, just changed it